src/docfx/restore/lock/DependencyLockModel.cs

```diff
@@ -8,7 +8,5 @@ namespace Microsoft.Docs.Build
     internal class DependencyLockModel : DependencyVersion
```

Comment:
also remove `DependencyVersion.Hash`, then it feels `DependencyVersion` could be consolidated into `DependencyLockModel`

Reply:
Also we don't need to calculate hash in  https://github.com/OsmondJiang/docfx/blob/3cd291a04546a14956fd9fabd19689a80b9ff135/src/docfx/restore/RestoreFile.cs#L53

Reply:
agree with your first point, for the second, we need keep that until we change the file restore logic(always restore to one place for one url)